signer/src/message.rs

```diff
@@ -224,16 +224,70 @@ pub struct BitcoinPreSignRequest {
 #[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
 pub struct BitcoinPreSignAck;
 
+/// The identifier for a WSTS message.
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum WstsMessageId {
+    /// The WSTS message is related to a Bitcoin transaction signing round.
+    BitcoinTxid(bitcoin::Txid),
+    /// The WSTS message is related to a rotate key verification operation.
+    RotateKey(PublicKey),
```

Comment:
Tempted to rename the whole `RotateKey` thing to `DkgVerify` or something, thoughts?

Reply:
+1 on that, it avoid misunderstandings

Reply:
I gave them some new names in [5f99a29](https://github.com/stacks-network/sbtc/pull/1287/commits/5f99a29c7101d87b2a0ab5c44e2c829c6e2d621c) which I think provide more context.

Again, these aren't used specifically in this PR -- control flow based on them is introduced in #1285.

Reply:
It's not the best, but it, like the others, now say exactly what they're used for (we don't sign any other kinds bitcoin transactions).

This field went away completely in the message id refactor i was playing with this morning, so when we revisit this I'm sure we'll sort it out.